code/datums/uplink_item.dm

```diff
@@ -1638,6 +1638,15 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/storage/backpack/duffel/syndie/bulldogbundle
 	cost = 9 // normally 12
 	gamemodes = list(/datum/game_mode/nuclear)
+	
+/datum/uplink_item/bundles_TC/ogre12
+	name = "Ogre12 Bundle"
+	desc = "Fat and mean: For people who want to assault and clear rooms. Contains the obscure \
+			Ogre 12 shotgun, two 16 round drum mags, and two extra box mags."
+	reference = "BULO"
+	item = /obj/item/storage/backpack/duffel/syndie/ogre12bundle
+	cost = 12 // normally 18 2(slg in gun) + (2drm + 2slg + 2slg) + (2drm + 2buck + 2buck) + 2buck + 2drg
```

Comment:
This is way too cheap for the amount of ammo you get with it

Reply:
I was going to make a new bundle with just three drums. It would lower the mag count by one with no gun. The bulldog ammo bundle is 12TC and comes with 9 mags. If you buy the drum bundle you get 3 fewer mags but they're all in drums. If you buy the ammo bundle you would have more rounds to fire.